public/gitinfo.php

```diff
@@ -16,17 +16,18 @@
 
 include dirname(__DIR__) . '/config/config.php';
 require_once 'public/login.php';
+require_once 'models/user.php';
 
-function echo_svn_output($cmd)
+function echo_git_output($cmd)
 {
```

Comment:
It seems a bit of this refactoring is exploding the size of this PR and unrelated to password hashing (making it harder to review).